save-demo/src/main/kotlin/com/saveourtool/save/demo/config/ConfigProperties.kt

```diff
@@ -9,12 +9,44 @@ import org.springframework.boot.context.properties.ConfigurationProperties
 import org.springframework.boot.context.properties.ConstructorBinding
 
 /**
+ * @property backendUrl URL of backend
+ * @property agentConfig configuration of save-demo-agents that are run by save-demo
  * @property s3Storage configuration of S3 storage
- * @property backend
+ * @property kubernetes kubernetes configuration
  */
 @ConstructorBinding
 @ConfigurationProperties(prefix = "demo")
 data class ConfigProperties(
+    val backendUrl: String,
+    val agentConfig: AgentConfig,
     val s3Storage: S3OperationsProperties,
-    val backend: String,
+    val kubernetes: KubernetesConfig,
+)
+
+/**
+ * @property demoUrl url of save-demo
+ */
+data class AgentConfig(
+    val demoUrl: String
+)
+
+/**
+ * @property apiServerUrl
+ * @property serviceAccount
+ * @property namespace
+ * @property useGvisor
+ * @property agentCpuRequests
+ * @property agentCpuLimits
+ * @property agentMemoryRequests
+ * @property agentMemoryLimits
+ */
+data class KubernetesConfig(
+    val apiServerUrl: String,
+    val serviceAccount: String,
+    val namespace: String,
+    val useGvisor: Boolean,
+    val agentCpuRequests: String = "100m",
```

Comment:
Can you please explain the conventions in the KDoc?

What do `10m` and `10Mi` actually represent?

Reply:
`m` stands for milli, `M` stands for Mega.
By default, `M` and `m` are powers of 10.
To be more accurate and use `M` as 1024 instead of 1000, `i` should be provided: `Mi`
https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/

Also added this info into kdoc